src/lib/pluginUtils.ts

```diff
@@ -66,6 +66,16 @@ export const PROPOSAL_PLUGIN_NAMES = [
   "ContributionRewardExt",
 ];
 
+export const PLUGINS_UI_NAMES = {
```

Comment:
do we do not have these name somewhere else in the code?

Reply:
**Short answer:** no
**Long answer:** 
There is a function: `pluginName(plugin: IPluginState|IContractInfo, fallback?: string)` in `pluginUtils.ts` that returns the display name. This function is used in the `plugin to remove`  tab but there you have the expected params of the function (`IPluginState|IContractInfo`).
In `plugin to add` we don't have these variables unless we make a bigger change in the code there.
In addition I don't think this function does exactly what we need (for example: for `GenericScheme` the result may be `Blockchain Interaction`).
Baside that there is no other place in the code with the display names.